### PR TITLE
update entity and h2 with new columns

### PR DESCRIPTION
**Pull request by eclendenning-va**

### CODE REVIEW

**Comment by jhulbert-va** [comment]
This boi will need to merge first: https://github.com/department-of-veterans-affairs/health-apis-datamart-synthetic-records/pull/129

**Review by jhulbert-va — DISMISSED** [review]

**Comment by jhulbert-va** [comment]
Merging this will also go unstable until https://github.com/department-of-veterans-affairs/health-apis-data-query/pull/718 merges (vulcan will be auto-upgraded and some of the vulcan methods will have been deprecated).

**Comment by jblefkove-va** [comment]
There will be a merge conflict on `mitre.mv.db` between this and https://github.com/department-of-veterans-affairs/health-apis-data-query/pull/719

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/condition/ConditionEntity.java [hunk 1]
```diff
@@ -24,6 +25,8 @@
  * <pre>
  * CREATE TABLE [app].[Condition](
  *  [CDWId] [varchar](50) NOT NULL,
+ *  [CdwIdNumber] [bigint] NOT NULL,
+ *  [CdwIdResourceCode] [char](1) NULL,
```
Comment (jblefkove-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/condition/ConditionEntity.java:29]: I think this block comment is redundant and should be removed

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/condition/ConditionEntity.java [hunk 2]
```diff
@@ -45,6 +27,14 @@
 @EqualsAndHashCode(onlyExplicitlyIncluded = true)
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 public class ConditionEntity implements DatamartEntity {
+  @Column(name = "CdwIdNumber")
+  @EqualsAndHashCode.Include
```
Comment (bschofield-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/condition/ConditionEntity.java:31]: Should not be part of equals and hash code until these become the primary key.

Comment (jblefkove-va) [reply data-query/src/main/java/gov/va/api/health/dataquery/service/controller/condition/ConditionEntity.java:31]: It's permissible, but I think it makes more sense not to switch over hashcode and equals until these new columns are officially marked `@Id`

**Review by jhulbert-va — APPROVED** [review]

**Pull request merged**